community/chef-infra-server.mql.yaml

```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Tim Smith
         email: tim@mondoo.com
+    tags:
+      mondoo.com/category: security
+      mondoo.com/platform: linux,unix
```

Comment:
This one in linux only though. Sorry

Reply:
Then the asset filter is wrong too

Reply:
Updated the tag and also fixed the asset filter